src/sql/workbench/services/notebook/browser/models/notebookModel.ts

```diff
@@ -652,7 +653,7 @@ export class NotebookModel extends Disposable implements INotebookModel {
 	}
 
 	private isValidConnection(profile: IConnectionProfile | connection.Connection) {
-		if (this._standardKernels) {
+		if (this._standardKernels.length > 0) {
```

Comment:
Do you even need these if checks anymore? If there aren't any standardKernels then it looks like it should just fail to find and result in the same behavior.

Reply:
Yep I can remove the checks